roles/import-from-url/tasks/deprovision.yml

```diff
@@ -2,35 +2,8 @@
 - name: Change project to {{ namespace }}
   command: "oc project {{ namespace }}"
 
-- name: Target is VM?
-  set_fact:
-    image_type: vm
-  when: _apb_plan_id == 'url'
-
-- name: Target is Template?
-  set_fact:
-    image_type: template
-  when: _apb_plan_id == 'url-template'
-
-- name: Set unused template variables
-  set_fact:
-    size_bytes: 0
-    pvc_name: "{{ image_type }}-{{ vm_name }}-disk-01"
-    disk_bus: "{{ disk_bus_by_os_type[os_type] }}"
-    disk_size_bytes: 0
-
-- name: Build VM resources
-  template:
-    src: "{{ role_path }}/../../templates/{{ os_type }}-{{ image_type }}.j2"
-    dest: /tmp/vm-resources.yml
-
 - name: Delete VM resources
-  command: "oc delete -f /tmp/vm-resources.yml --ignore-not-found=true"
-
-- name: Build PVC
-  template:
-    src: cdi-pvc.yml
-    dest: /tmp/pvc.yml
+  command: "oc delete vm -l import-vm-apb/transaction_id={{ _apb_service_instance_id }} --ignore-not-found=true"
```

Comment:
Sometimes this will be a virtual machine and sometimes a template.

Reply:
Good point, let me fix that.